### ISSUE TITLE
Add to editorconfig new variable with url of the page where editor command was executed

### ISSUE THREAD

**Issue by jumper047** [tridactyl/tridactyl#4625]
Idea is to use that information to set up editor settings according to page where it was summoned. Some sites uses plain text, some sites - markdown, you can even write your own plugin for your favorite site if it uses some special syntax.

Alternate idea - to give tridactyl user more control on tmp file's name, so he can set up in config to create, for example,  tmp file names with name like "github_com_%random%_.md" for github.

**Comment by bovine3dom** [comment]
I don't have any issue with adding a variable to `editorcmd`, but we do already include the hostname in the filename for this exact reason - 

https://github.com/tridactyl/tridactyl/blob/12fcc25efebc354b80ebc0244ce54bb0329b5dc6/src/excmds.ts#L363

E.g. the filename for this text field as I edit it now is `tmp_github.com_8ecm7gyv.txt`. In Vim you can set rules based on this with an `autocmd BufRead` - https://stackoverflow.com/questions/11635710/setting-vim-options-depending-on-file-name#answer-11635742

**Comment by bovine3dom** [comment]
so for your use case you could add `autocmd BufRead tmp_github.com_* setlocal filetype=markdown` to your `.vimrc`

**Comment by jumper047** [comment]
Yep, I suppose filename is enough for cases I have now. I thought separate variable could be useful for some imaginary case when some site has different input styles on different urls. Also IMHO variable with clear purpose is better than relying on tmp filename which can be changed for some reason. But if you don't see some value in it for now, then, probably, it should be postponed until first real case.

> so for your use case you could add `autocmd BufRead tmp_github.com_* setlocal filetype=markdown` to your `.vimrc`

`init.el`, I'm those guy who made config with emacs-like bindings for tridactyl:)

**Comment by kohane27** [comment]
I have the following line in `tridactylrc`:

```
set editorcmd /usr/bin/kitty -e /usr/bin/nvim -c "set filetype=markdown"
```

I use kitty and I can set the filetype here.
